src/Symfony/Bridge/Doctrine/Tests/Messenger/DoctrinePingConnectionMiddlewareTest.php

```diff
@@ -103,9 +103,12 @@ public function testInvalidEntityManagerThrowsException()
 
     public function testMiddlewareNoPingInNonWorkerContext()
     {
-        $this->connection->expects($this->never())
-            ->method('ping')
-            ->willReturn(false);
+        // This method has been removed in DBAL 3.0
```

Comment:
What about moving this comment above the test method declaration?

Reply:
Because I'm documenting this `if` statement only.